### PR TITLE
Changing Glance Column Width default to fill card

### PR DESCRIPTION
**Pull request by zsarnett**
This defaults the column width if the entities supplied is less than 5 to a percentage that fills the container of the card.

From this: 

Before

To this:

After

I believe this could lead to not needing the `column_width` variable. Easier to configure.

### CODE REVIEW

**Comment by balloob** [comment]
I don't think that it looks nice if you have 2 glance cards above each other, one with 4 and the other with 3 entities. Now they won't be aligned.

**Comment by zsarnett** [comment]
I feel like that's a better example for when to use the variable

**Comment by zsarnett** [comment]
I feel like it looks more broken to have them squished together only taking up 60% of the width by default. How many people have two glance cards one on top the other

**Comment by zsarnett** [comment]
Example 1
Example1

Example 2
Example2

Example3

Example3

Example 4

Example4

**Comment by armills** [comment]
I personally prefer this to the old behavior.

**Comment by cgarwood** [comment]
I agree, I like this behavior better than having them fixed and left-aligned.

**Comment by bramkragten** [comment]
In favor of the new!

**Comment by noxhirsch** [comment]
The new looks way better 👍

**Comment by zsarnett** [comment]
> More than five entities will still show up on the next row, and be left aligned, right?

This is Correct

File: src/panels/lovelace/cards/hui-glance-card.ts [hunk 1]
```diff
@@ -65,10 +65,10 @@ export class HuiGlanceCard extends HassLocalizeLitMixin(LitElement)
       }
     }
 
-    this.style.setProperty(
-      "--glance-column-width",
-      config.column_width || "20%"
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-glance-card.ts]: Remove from interface.

**Review by balloob — APPROVED** [review]

**Comment by benleb** [comment]
Ok, this means the simple way to use more than 5 entities in one row is gone?! Really!?

**Pull request merged**